devsetup/tripleo/overcloud_roles.yaml

```diff
@@ -225,14 +225,17 @@
     - OS::TripleO::Services::BootParams
     - OS::TripleO::Services::CACerts
     - OS::TripleO::Services::Clustercheck
+    - OS::TripleO::Services::Collectd
```

Comment:
are these only relevant for the mutli cell job ? If yes then perhaps we need to consider a mutli_cell_overcloud_roles.yaml for that specific case? 

Otherwise can you please ensure to run the ceph/noceph "standard" multinode job at least once when testing this to sanity check we don't need to fix something else there to enable these

Reply:
Those are default for tripleo for cellcontroller role